tests/end_of_file_fixer_test.py

```diff
@@ -15,6 +15,8 @@
     (b'foo', 1, b'foo\n'),
     (b'foo\n\n\n', 1, b'foo\n'),
     (b'\xe2\x98\x83', 1, b'\xe2\x98\x83\n'),
+    (b'foo\r\n', 1, b'foo\n'),
+    (b'foo\r\n\r\n', 1, b'foo\n'),
```

Comment:
hmmm, it _would_ be better if it kept the `\r\n` at the end -- is it difficult to adjust the logic to do that?

Reply:
Thanks for the quick response on this. I'll get it updated.